### DISCUSSION TITLE
recovery for gamil

### DISCUSSION THREAD

**Discussion by Ericgost** [community/community#144729] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### GitHub Feature Area

Security and Privacy

### Body

recovery for gamil account

**Comment by queenofcorgis** [comment]
Hi @Ericgost ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the the [Gmail Community](https://support.google.com/mail/community?hl=en). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by Ericgost** [comment]
thank you for Ur reply,yes I need help to recovering my personal
Gmail..what I need to give for next step going ..too
